.github/workflows/main.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       fail-fast: false
       matrix:
         node-version: [16.14.2]
-        java-version: ["8", "11", "17", "19"]
+        java-version: ["8", "11", "17", "19", "20", "21"]
```

Comment:
We can just test towards LTS and the latest version. So that could be: `8`, `11`, `17`, `21`